custom/enikshay/templates/enikshay/partials/report_content.html

```diff
@@ -0,0 +1,98 @@
+{% load i18n %}
+
+{% block pretable %}
+    {% if charts %}
+    <div class="row">
+        {% for chart in charts %}
+        <div id='chart_{{ report.slug }}_{{ forloop.counter }}' class="col-md-{{ chart_span }} collapse">
+            {% if chart.title %}<h4 style="text-align: center;">{{ chart.title }}</h4>{% endif %}
+            <svg style='height: {{ chart.height }}px'> </svg>
+        </div>
+        {% endfor %}
+    </div>
+    {% endif %}
+{% endblock pretable %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h2 class="panel-title">
+        {% if not report.needs_filters %}
+            {{ report.report_title }}
+            {% for subtitle in report.report_subtitles %}
+            <small>{{ subtitle }}</small>
+            {% endfor %}
+        {% else %}
+            {% trans "Please apply filters to view:" %} {{ report.report_title }}
+        {% endif %}
+        </h2>
+    </div>
+    <div class="panel-body-datatable">
+    {% block reporttable %}
+    {% if report.needs_filters %}
+        {% include 'reports/partials/description.html' %}
+    {% else %}
```

Comment:
is there a reason why none of this could be pulled out/shared with [tabular.html](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/async/tabular.html#L73) ?